slides/src/content/routing.md

```diff
@@ -113,43 +114,70 @@ For example the following code defines a link to the route at path `component-on
 <a [routerLink]="['/component-one']">Component One</a>
 ```
 
-Alternatively, you can navigate to a route by calling the `navigate` function on the router:
+Alternatively, you can navigate to a route by calling the `navigate` function on the router with the path array as the argument. For example to navigate to component-one, we could use:
 
-```javascript
-this.router.navigate(['/component-one']);
+```ts
+var path = ['/component-one'];
+router.navigate(path);
```

Comment:
If this is meant to be Typescript, I think it should be:

`const path = ['/component-one'];`